.github/workflows/weekly-enviroment.yml

```diff
@@ -81,6 +84,21 @@ jobs:
           # Apply the manifest file
           ssh -o StrictHostKeyChecking=no -v -i weekly-key.pem "ubuntu@${{ secrets.WEEKLY_ENVIRONMENT_EC2_PUBLIC_IP }}" "kubectl apply -f manifest.yaml"
 
+      - name: Check if the latest index was updated
+        working-directory: ${{ env.HC_DIR }}
+        env:
+          max_attempts: 10
+          retry_wait_seconds: 60
+        run: |
+          ./count_latest_index_result.sh ${{ env.ELASTICSEARCH_URL }} ${{ env.ELASTICSEARCH_PASSWORD }} ${{ env.max_attempts }} ${{ env.retry_wait_seconds }}
+
```

Comment:
What will happen if "count_latest_index_result" script return failure? The workflow does not contain cleanup steps. Is it planned to add also environment cleanup?